organization/contact-details.md

```diff
@@ -0,0 +1,29 @@
+---
+type: Resource
+---
+
+## Contact us
+
+Our mission requires collaboration! We're eager for your questions or suggestions.
+
++ email: <info@publiccode.net>
++ post: Keizersgracht 617, 1017 DS, Amsterdam, the Netherlands
++ telephone: +31 6 17 96 02 05 (Boris van Hoytema's phone)
```

Comment:
Perhaps this can be 'you can reach the chief executive Boris van Hoytema on ...'